test

```diff
@@ -18,16 +18,9 @@ export src=$(dirname $(python -c "import $PROJECT as p; print p.__file__"))
 # See: http://nedbatchelder.com/code/coverage/subprocess.html
 python $TOP/tests/testing/install_coverage_pth.py
 
-
-# careful not to leave .coverage.$HOST.$RANDOM files laying about on failure.
-combine() { coverage combine --rcfile=$TOP/.coveragerc; }
-trap combine ERR
-
-combine
+coverage combine --rcfile=$TOP/.coveragerc; 
 coverage erase --rcfile=$TOP/.coveragerc
 COVERAGE_PROCESS_START=$TOP/.coveragerc py.test "$@" $TOP/tests $src
-combine
+coverage combine --rcfile=$TOP/.coveragerc; 
 coverage html --rcfile=$TOP/.coveragerc
 coverage report --rcfile=$TOP/.coveragerc
```

Comment:
@bukzor I believe this was your fix that you ended with, at any rate i'm not sure what you wanted to do with this file, I am tempted on leaving it out of this review.